lib/repl.js

```diff
@@ -763,6 +763,8 @@ exports.start = function(prompt,
                          useGlobal,
                          ignoreUndefined,
                          replMode) {
+  console.log(`Welcome to Node.js ${process.version}.\n` +
+    'Type ".help" for more information.');
```

Comment:
Given that `repl` is used by userland in some cases, and given that our docs tell users to use `start()` to create `REPLServer` instances, this message would end up being printed for userland uses also, which does not seem appropriate.

Reply:
Any suggestion how do I make the related change ?

Reply:
James is correct and I believe the correct place to do so would be in https://github.com/nodejs/node/blob/b4e670dc26b50271457a257ccb244b14ff323b64/lib/internal/repl.js#L22-L70

Reply:
I have moved to the new location.